tox.ini

```diff
@@ -5,8 +5,9 @@ envlist = flake8, django15, django16, django17, django18
 [testenv]
 setenv =
     DJANGO_SETTINGS_MODULE=test_settings
-    PYTHONPATH = {toxinidir}:{toxinidir}/ajax_select
+    PYTHONPATH = {toxinidir}:{toxinidir}/ajax_select:{toxinidir}/tests
```

Comment:
oh, its me that is confused. I was working on moving the tests inside the package but I didn't finish doing it. they are moved and staged only in my working copy. no worries, keep it just as you have it.